styleguide/_layouts/default.jade

```diff
@@ -31,50 +31,48 @@ html(lang="en")
             a(href="/").navZoolander-link Welcome
 
           li.navZoolander-item
-            a(href="#").navZoolander-link.navZoolander-hasDropdown Global Brand
+            a(href="#").navZoolander-link.navZoolander-hasDropdown Rackspace Brand Guidelines
             ul.navZoolander-dropdown
               li.navZoolander-item
-                 a(href="global/").navZoolander-dropdownLink Overview
+                a(href="global/").navZoolander-dropdownLink Overview
               li.navZoolander-item
-                 a(href="global/brand-elements").navZoolander-dropdownLink Brand Elements
-
-          li.navZoolander-item
-            a(href="global/voice").navZoolander-link Voice and Tone
-
-          li.navZoolander-item
-            a(href="global/components").navZoolander-link Components
+                a(href="global/brand-elements").navZoolander-dropdownLink Brand Elements
+              li.navZoolander-item
+                a(href="global/voice").navZoolander-dropdownLink Voice and Tone
 
           li.navZoolander-item
-            a(href="#").navZoolander-link.navZoolander-hasDropdown Assets/Elements
+            a(href="#").navZoolander-link.navZoolander-hasDropdown Zoolander Global Elements
             ul.navZoolander-dropdown
               li.navZoolander-dropdownItem
                 a(href="derek/assets/").navZoolander-dropdownLink Overview
               li.navZoolander-dropdownItem
-                a(href="derek/assets/icon-font").navZoolander-dropdownLink Icon Font
-              li.navZoolander-dropdownItem
-                a(href="derek/assets/forms").navZoolander-dropdownLink Forms
-              li.navZoolander-dropdownItem
-                a(href="derek/assets/buttons").navZoolander-dropdownLink Buttons/CTAs
-              li.navZoolander-dropdownItem
                 a(href="derek/assets/typography").navZoolander-dropdownLink Typography
               li.navZoolander-dropdownItem
                 a(href="derek/assets/photography").navZoolander-dropdownLink Photography/Banners
               li.navZoolander-dropdownItem
-                a(href="derek/assets/icon-font").navZoolander-dropdownLink Tabs
+                a(href="derek/assets/swatches").navZoolander-dropdownLink Colors/Swatches
```

Comment:
We probably shouldn't worry about it in this PR, but for future PR purposes:

Does it make sense to have Global examples under the `/derek/` URL path, if there's a separate set of navigation for the Derek Theme itself.  Should they be under `/global/` instead? That has potential to cause confusion.  Once again, not a question to get addressed in this PR, but worth thinking about.